src/build.d

```diff
@@ -1444,6 +1449,30 @@ class Dependency
             }
         }
     }
+
+    /// Writes relevant informations about this dependency to stdout
+    private void dump()
+    {
+        scope writer = stdout.lockingTextWriter;
```

Comment:
For another PR, could use `lockingTextWriter` to make sure the `dryRun` output isn't mangled between multiple threads.

Reply:
See #10622